descwl_coadd/coadd.py

```diff
@@ -134,11 +142,13 @@ def _make_exps(self):
 
                 weight = se_obs.weight.array
 
-                psf_image = se_obs.get_psf(
+                psf_gsimage, psf_offset = se_obs.get_psf(
                     pos.x,
                     pos.y,
-                    center_psf=True,
-                ).array
+                    center_psf=False,
+                    get_offset=True,
```

Comment:
So this now matches what we are doing in pizza cutter, right?

Reply:
Yes